event_socket_server/base_server.py

```diff
@@ -27,6 +30,13 @@ def __init__(self, time, func, args, kwargs):
         self.cancel = False
         self.dispatched = False
 
+    def __eq__(a, b):
+        return a.time == b.time and a.func == b.func and a.args == b.args and a.kwargs == b.kwargs and \
```

Comment:
What a genius. Dude, how the fuck is this `__eq__` supposed to match up with `__lt__`??? Remember the invariant `a == b`  ⇔ `!(a < b) && !(b < a)`...